doc/dftb+/manual/dftbp.tex

```diff
@@ -2539,10 +2543,10 @@ \section{Options}
   value can be used to reproduce earlier MD calculations by setting
   the initial seed to the same value.)
 
-\item[\is{MinimiseMemoryUsage}] Tries to minimise memory usage by
```

Comment:
Parser is still reading this option, but initprogram ignores it. Should this be described as a removed feature in the manual?

Reply:
Not sure, as we wanted to replace the eigenvector storage via disc with on-the-fly recalculation. I added a warning, which is printed if the users turns it on.